05 - Flex Panel Gallery/index-START.html

```diff
@@ -105,10 +125,22 @@
   </div>
 
   <script>
+    //// selectors
+    const panels = document.querySelectorAll('.panel');
 
-  </script>
-
+    //// callbacks
+    const handleOpenPanel = function () {
```

Comment:
why not `function handleOpenPanel() { ...`?

I defining a function to assign it to a variable is a little cumbersome

Reply:
I got a bit too wild in my functional experimentation. all back to good-practice!